test_scripts/Polices/App_Permissions/ATF_HP_User_Consent_NO.lua

```diff
@@ -0,0 +1,309 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- On getting negative user consent from HMI and storage this data into LocalPolicyTable, SDL must:
+-- 1. assign only the permissions for which there is user consent in the local policy table and permissions (if any) that are listed in 'pre_DataConsent' section (that is, 'default_hmi', 'groups' and other) for this application 
+-- 2. notify HMI via onPermissionsChange() notification about the applied permissions.
+-- 3. notify mobile app via onPermissionsChange() notification about the applied permissions.
+-- Preconditions:
```

Comment:
Please check template how sections are organized. https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32 
Description:
Put here the short description
1. Used preconditions 
Put here the preconditions
2. Performed steps 
Only here list what actions are done 
Expected result:
Put here what are the expectations